openIt/ngsild-payloads/authorization.jsonld

```diff
@@ -78,5 +78,17 @@
   "generatedUntil": {
     "type": "Property",
     "value": "2024-09-02T23:59:59.000Z"
-  }
+  },
+  "viewedBy": [
```

Comment:
hmm, no, if you have more than one instance, having a datasetId is required.

in your case, you can just have:

```
"viewedBy": {
      "type": "Relationship",
      "object": "urn:ngsi-ld:UserProfile:123",
      "observedAt": "2024-11-25T09:01:00Z"
}
```

Reply:
But it will most likely have more than one instance since multiple users could have access to the code or even the same person viewing the code many different times. So I should add back in the datasetId then?